docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
   mysql:
     image: mysql:5.7
     ports:
-    - "3306:3306"
+    - "3309:3306"
```

Comment:
Can you undo this please.

Reply:
Fixed, good catch. Had to yeet this for my windows machine sorry